web/app/scripts/details/details-constants-partial.html

```diff
@@ -0,0 +1,38 @@
+<div class="col-sm-6">
+    <label>
+        Occurred
+    </label>
+    <div class="value constant date occurred">
+        {{ ::ctl.record.occurred_from | date : ctl.dateFormat }}
+    </div>
+</div>
+<div class="col-sm-6">
+    <label>
+        Created
+    </label>
+    <div class="value constant date created">
+        {{ ::ctl.record.created | date: ctl.dateFormat }}
+    </div>
+</div>
+<div class="col-sm-6">
+    <label>
+        Latitude
+    </label>
+    <div class="value constant float latitude">
+        {{ ::ctl.record.geom.coordinates[1] }}
+    </div>
+</div>
+<div class="col-sm-6">
+    <label>
+        Longitude
+    </label>
+    <div class="value constant float longitude">
+        {{ ::ctl.record.geom.coordinates[0] }}
+    </div>
+</div>
+<div class="col-md-12">
+    <div class="map" leaflet-map driver-embed-map
+        lat="{{ ::ctl.record.geom.coordinates[1] }}"
```

Comment:
It doesn't appear to be possible to have them passed as numbers. I tried removing the quotes and removing the colons, but it always comes across as a string. It does still work though as-is, so probably not a problem. If you know of an alternate way, let me know.